src/style.css

```diff
@@ -1 +1,115 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.search__container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 20px 8px;
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+}
+
+.search-bar__input {
+  display: block;
+  width: 100%;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+}
+
+.search-bar__input--big {
+  height: 70px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  padding-left: 62px;
+  background-image: url('/src/images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: 25px center;
+  font-size: 16px;
+  color: rgba(61, 78, 97, 1);
+  font-weight: bold;
+}
+
+.search-bar__input--big::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-weight: normal;
+}
+
+.search-bar__input--big:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__input--big:focus {
+  outline: none;
+  background-image: url('/src/images/Search.svg'),
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat;
+  background-position:
+    25px center,
+    0 0;
+  background-size:
+    19px 19px,
+    100% 100%;
+  text-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__input--small {
+  height: 42px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  box-sizing: border-box;
+  padding-left: 33px;
+  background-image: url('/src/images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: 12px center;
+  background-size: 11px 11px;
+  color: rgba(61, 78, 97, 1);
```

Comment:
Similar to the big search bar, consider adding a fallback font stack for the small search bar to ensure text is displayed correctly across different browsers.